ws-nextjs-app/tsconfig.json

```diff
@@ -11,6 +11,7 @@
     "next-env.d.ts",
     "**/*.ts",
     "**/*.tsx",
+    "../src/**/*.d.ts",
```

Comment:
Hello, could someone let me know if this is okay?

Reply:
Looks good, me and Aaron chatted about this as i had the same errors with the typing - this is the best way to let it use the express app type definitions!